input/fsh/codeSystem/codeSystem.fsh

```diff
@@ -0,0 +1,1319 @@
+* #161364 ^designation[0].language = #en
+* #161364 ^designation[=].value = "Lamivudine / tenofovir disoproxil"
+* #161364 ^property[0].code = #conceptclass
+* #161364 ^property[=].valueString = "Misc"
+* #161364 ^property[+].code = #datatype
+* #161364 ^property[=].valueString = "N/A"
+* #1652 "Lamivudine / nevirapine / zidovudine"
+* #1652 ^designation[0].language = #en
+* #1652 ^designation[=].value = "Lamivudine / nevirapine / zidovudine"
+* #1652 ^property[0].code = #conceptclass
+* #1652 ^property[=].valueString = "Misc"
+* #1652 ^property[+].code = #datatype
+* #1652 ^property[=].valueString = "N/A"
+* #5424 "Other antiretroviral drug"
+* #5424 ^designation[0].language = #en
+* #5424 ^designation[=].value = "Other antiretroviral drug"
+* #5424 ^property[0].code = #conceptclass
+* #5424 ^property[=].valueString = "Misc"
+* #5424 ^property[+].code = #datatype
+* #5424 ^property[=].valueString = "N/A"
+* #5811 "Unknown antiretroviral drug"
+* #5811 ^designation[0].language = #en
+* #5811 ^designation[=].value = "Unknown antiretroviral drug"
+* #5811 ^property[0].code = #conceptclass
+* #5811 ^property[=].valueString = "Misc"
+* #5811 ^property[+].code = #datatype
+* #5811 ^property[=].valueString = "N/A"
+* #630 "Lamivudine / zidovudine"
+* #630 ^designation[0].language = #en
+* #630 ^designation[=].value = "Lamivudine / zidovudine"
+* #630 ^property[0].code = #conceptclass
+* #630 ^property[=].valueString = "Misc"
+* #630 ^property[+].code = #datatype
+* #630 ^property[=].valueString = "N/A"
+* #103166 "Abacavir / lamivudine"
+* #103166 ^designation[0].language = #en
+* #103166 ^designation[=].value = "Abacavir / lamivudine"
+* #103166 ^property[0].code = #conceptclass
+* #103166 ^property[=].valueString = "Misc"
+* #103166 ^property[+].code = #datatype
+* #103166 ^property[=].valueString = "N/A"
+* #70056 "Abacavir"
+* #70056 ^designation[0].language = #en
+* #70056 ^designation[=].value = "Abacavir"
+* #70056 ^property[0].code = #conceptclass
+* #70056 ^property[=].valueString = "Misc"
+* #70056 ^property[+].code = #datatype
+* #70056 ^property[=].valueString = "N/A"
+* #71647 "Atazanavir"
+* #71647 ^designation[0].language = #en
+* #71647 ^designation[=].value = "Atazanavir"
+* #71647 ^property[0].code = #conceptclass
+* #71647 ^property[=].valueString = "Misc"
+* #71647 ^property[+].code = #datatype
+* #71647 ^property[=].valueString = "N/A"
+* #74807 "Didanosine"
+* #74807 ^designation[0].language = #en
+* #74807 ^designation[=].value = "Didanosine"
+* #74807 ^property[0].code = #conceptclass
+* #74807 ^property[=].valueString = "Misc"
+* #74807 ^property[+].code = #datatype
+* #74807 ^property[=].valueString = "N/A"
+* #75523 "Efavirenz"
+* #75523 ^designation[0].language = #en
+* #75523 ^designation[=].value = "Efavirenz"
+* #75523 ^property[0].code = #conceptclass
+* #75523 ^property[=].valueString = "Misc"
+* #75523 ^property[+].code = #datatype
+* #75523 ^property[=].valueString = "N/A"
+* #75628 "Efavirenz"
```

Comment:
Wrong display name, at least according to CIEL. Display should be "Emtricitabine" ([CIEL Link](https://app.openconceptlab.org/#/orgs/CIEL/sources/CIEL/concepts/75628/))